src/endpoints/query/impl.cpp

```diff
@@ -261,10 +261,20 @@ namespace
 								options |= UPPER_CASE_WHERE;
 								boost::algorithm::to_upper(query_iter->second);
 							}
+							else if (iter->second != "1") {
+								log::error("{}: Invalid value for [case-sensitive] parameter. Expected a 1 or 0.", fn);
```

Comment:
want to log the invalid value used?  not sure on this...

Reply:
I'm going to do a pass over all log messages in the future.

I'll create an issue for this.

Reply:
Created #157 for this.